vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/extensions.go

```diff
@@ -33,7 +33,9 @@ func startDaemonSetController(ctx ControllerContext) (bool, error) {
 	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "daemonsets"}] {
 		return false, nil
 	}
-	dsc, err := daemon.NewDaemonSetsController(
+	dsc, err := daemon.NewNodeSelectorAwareDaemonSetsController(
```

Comment:
@deads2k could we have a test that this is actually plugged-in and stays that way? considering we don't have an integration one

Reply:
> @deads2k could we have a test that this is actually plugged-in and stays that way? considering we don't have an integration one

I commented back here: https://github.com/openshift/origin/pull/18989#issuecomment-373747058 the node dependency actually makes it impractical to add integration tests for it.